### ISSUE TITLE
JupyterCon 2023 talk

### ISSUE THREAD

**Issue by manics** [jupyterhub/team-compass#630]
I've just found an email sent on 21 Feb to the jupyterhub-team-council email in my spam folder, inviting JupyterHub to give a 25 min talk at JupyterCon. There are no replies- did anyone else see the email or did it also end up in your spam?

**Comment by consideRatio** [comment]
I saw and forgot about it, are you @manics or someone else willing to present or co-present a talk?

Btw @manics are you coming to JupyterCon?! It would be amazing to meet you in person finally!

**Comment by sgibson91** [comment]
I saw it too, but I have talks already so didn't volunteer.

**Comment by manics** [comment]
I'm not.

**Comment by minrk** [comment]
I missed it somehow. I'm going and can give the talk. Happy to have anyone else co-present, too.

**Comment by minrk** [comment]
I missed another thing, that organizers want an abstract **by Friday**, so I've started a draft folks can edit https://hackmd.io/@minrk/H10_ijxW2/edit

Will it be me and @consideRatio? I'm happy to leave room for someone else if they want to give it a go, but I'm also happy to do it if nobody wants to take it on.

It's a short slot, only 15 minutes, so I figure no more than 2 speakers and just a quick tour of what's new/coming that we'd specifically like to advertise, e.g.:

1. _extremely_ brief "what is JupyterHub?" for context
2. community topic (or covered elsewhere already?)
3. what's new in JupyterHub core
  - new: granular permissions, better RTC 
  - coming: sharing access
4. what's new/coming in z2jh/tljh (@consideRatio?)
5. what's new/coming in repo2docker/BinderHub/mybinder.org
6. misc, if needed (e.g. traefik)

**Comment by minrk** [comment]
OK, great! Sorry for the poor timing. I'll ping you for a chat to go over it one last time during the day tomorrow, if you're free.

**Comment by KirstieJane** [comment]
Hi @consideRatio and @minrk - did you give this talk? Is it available somewhere? If so then we could maybe add an action to this issue to add a link to the talk somewhere in this repository 😄 

If there isn't an artefact from 2 years ago then that's totally fine and we can close the issue I think ✨

**Comment by consideRatio** [comment]
Yes! Mostly a Min effort who presented alone with some assistance by me preparing things ahead of it!

**Comment by KirstieJane** [comment]
ok! So - in figuring out where to document this I found the https://compass.hub.jupyter.org/practices/external-communication page 🚀 🌟 

There's a shared google drive which is great and there are the links to discourse (https://discourse.jupyter.org/t/resources-for-giving-talks-about-the-jupyterhub-and-binder-projects/1685).

That page is focused on people who want to represent the community.... but what about content for folks who just want to learn about the project?

I'm wondering if a page in the team-compass with recordings would be useful? Or maybe not team compass but somewhere else?

(This isn't urgent - so no one feel they have to rush to an answer here - I thought I would be adding to an existing list but I'm not totally sure that's true....)
